tests/quarkus/src/test/java/io/hawt/tests/quarkus/suites/QuarkusSmokeTest.java

```diff
@@ -1,16 +0,0 @@
-package io.hawt.tests.quarkus.suites;
-
-import static io.cucumber.junit.platform.engine.Constants.FILTER_TAGS_PROPERTY_NAME;
-
-import org.junit.platform.suite.api.ConfigurationParameter;
-import org.junit.platform.suite.api.IncludeEngines;
-import org.junit.platform.suite.api.SelectClasspathResource;
-import org.junit.platform.suite.api.Suite;
-
-@Suite
-@IncludeEngines("cucumber")
-@SelectClasspathResource("features")
-@ConfigurationParameter(key = FILTER_TAGS_PROPERTY_NAME, value = "@quarkusSmokeTest")
-public class QuarkusSmokeTest {
```

Comment:
Whoops deleted this by accident when experimenting :D there shoudl be no issue with this